client/utils/validation.js

```diff
@@ -6,6 +6,14 @@ import { messages } from 'vee-validate/dist/locale/en.json';
 import some from 'lodash/some';
 import userApi from '@/api/user';
 
+const NAME_FORMAT_REGEX = /^[\p{Letter}\s'-.]+$/u;
+
+const nameFormat = {
+  validate: value => NAME_FORMAT_REGEX.test(value),
+  message: `The {_field_} field may only contain alphabetic characters, spaces,
+    dots (.), apostrophes (') and hyphens (-)`
```

Comment:
@malizeus Why dots 🤔

Reply:
Input names can be written in abbreviated forms (e.g Cuba Gooding Jr., J. K. Rowling ...) 😅